nx-X11/lib/xtrans/Xtransint.h

```diff
@@ -103,59 +103,6 @@ from The Open Group.
 #  include <sys/utsname.h>
 # endif
 
-/*
- * makedepend screws up on #undef OPEN_MAX, so we define a new symbol
- */
-
-# ifndef TRANS_OPEN_MAX
-
-#  ifndef X_NOT_POSIX
-#   ifdef _POSIX_SOURCE
-#    include <limits.h>
-#   else
-#    define _POSIX_SOURCE
-#    include <limits.h>
-#    undef _POSIX_SOURCE
-#   endif
-#  endif
-#  ifndef OPEN_MAX
-#   if defined(_SC_OPEN_MAX) && !defined(__UNIXOS2__)
-#    define OPEN_MAX (sysconf(_SC_OPEN_MAX))
-#   else
-#    ifdef SVR4
-#     define OPEN_MAX 256
-#    else
-#     include <sys/param.h>
-#     ifndef OPEN_MAX
-#      ifdef __OSF1__
-#       define OPEN_MAX 256
-#      else
-#       ifdef NOFILE
-#        define OPEN_MAX NOFILE
-#       else
-#        if !defined(__UNIXOS2__)
-#         define OPEN_MAX NOFILES_MAX
-#        else
-#         define OPEN_MAX 256
-#        endif
-#       endif
-#      endif
-#     endif
-#    endif
-#   endif
-#  endif
-#  if defined(_SC_OPEN_MAX)
-#   define TRANS_OPEN_MAX OPEN_MAX
-#  else /* !__GNU__ */
-#   if OPEN_MAX > 256
-#    define TRANS_OPEN_MAX 256
-#   else
-#    define TRANS_OPEN_MAX OPEN_MAX
-#   endif
-#  endif /*__GNU__*/
-
-# endif /* TRANS_OPEN_MAX */
-
```

Comment:
@uli42: please add Backported-to-NX-by: for f8cfe723e45834209d70ab0390f3fd782d6872fa.